src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Resources/xlf/Resources.de.xlf

```diff
@@ -61,6 +61,11 @@
         <target state="translated">Die Datei "{0}" ist nicht vorhanden.</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="AbortedMultiTestRunFinalization">
```

Comment:
>AbortedMultiTestRunFinalization [](start = 22, length = 31)

You'd probably want to ask for a special localization test pass for this or ask for a loc exception.